src/modules/Bookmark/routes.ts

```diff
@@ -0,0 +1,16 @@
+import { Router } from 'express';
+import bookmarkController from './controller.js';
+import auth from '../../middleware/auth.js';
+import validationSchema from './validation.js';
+import { validateRequest } from '../../middleware/validateRequest.js';
+
+const router = Router();
+
+router.post(
+  '/',
+  auth,
```

Comment:
for consistency with the rest of the routes, this should be named authMiddleware